tests/gps/test_create_beneficiary.py

```diff
@@ -21,6 +23,67 @@
 from tests.utils.test import KNOWN_SESSION_KEYS
 
 
+def helper_contains_nir_modal_gps(response, job_seeker, with_personal_information=True):
+    assertContains(
+        response,
+        f"""
+        <div class="modal-body">
+            <p>
+                Le numéro {format_nir(job_seeker.jobseeker_profile.nir)} est associé au compte de
+                <b>{mask_unless(job_seeker.get_full_name(), with_personal_information)}</b>.
+            </p>
+            <p>
+                Si ce n'est pas le bénéficiaire que vous souhaitez suivre,
+                cliquez sur « Suivre un autre bénéficiaire » afin de modifier le numéro de sécurité sociale.
+            </p>
+        </div>
+        <div class="modal-footer">
+            <button class="btn btn-sm btn-outline-primary" name="cancel" type="submit" value="1">
+            Suivre un autre bénéficiaire</button>
+            <button class="btn btn-sm btn-primary" name="confirm" type="submit" value="1">Continuer</button>
+        </div>
+        """,
+        html=True,
+    )
+
+
+def helper_contains_email_modal_gps(response, job_seeker, with_personal_information=True, nir_to_add=None):
```

Comment:
Cet argument n’est jamais passé.
```suggestion
def helper_contains_email_modal_gps(response, job_seeker, nir_to_add=None):
```